app/controllers/search_controller.rb

```diff
@@ -8,6 +8,7 @@ class SearchController < ApplicationController
   layout :determine_layout
 
   def index
+    @federation_enabled = federation_enabled?
```

Comment:
no need to put it here, directly call the helper in the view

Reply:
Done in https://github.com/ontoportal-lirmm/bioportal_web_ui/pull/780/commits/ab9f5b4e1d42adc165bbf5a92bf9dd49e102bf35